systest/scripts/conf/neutron-lbaas.tox.ini

```diff
@@ -6,13 +6,8 @@ skipsdist = True
 [testenv]
 setenv = VIRTUAL_ENV={envdir}
 usedevelop = True
-install_command =
-  {toxinidir}/tools/tox_install.sh {env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt} {opts} {packages}
```

Comment:
Handled in the neutronlbaas layer of the testrunner microservice in the labbench repo.